app/src/main/res/layout/bottom_sheet_new_automatch.xml

```diff
@@ -95,7 +95,6 @@
         android:id="@+id/searchButton"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        style="@style/BlueButton"
```

Comment:
This actually needs to be red and not blue. I know I introduced this as a bug in one of the recent versions, but can you please fix it as I am a bit confused right now at what colour is which :)

Reply:
so we want this to be "red" (the salmon-ish accent colour) on Light, but still Blue on Dark, right?

Reply:
Hmmm... I just realized we don't have the red colour anywhere in dark mode. It's up to you then. What do you think it's the best approach for dark mode here?

Reply:
I think it's ok to have a reduced palette in dark mode (it's difficult to match too many colors on a dark theme as well)
I made the Light theme button red again, and more importantly added a lighter blue to the text-on-dark-gray sections - the contrast wasn't good on the original PR